From 2fe9e102e0f6ba7cf1612de08369e813b5a8ec2d Mon Sep 17 00:00:00 2001
From: Serguei Sagalovitch <Serguei.Sagalovitch@amd.com>
Date: Fri, 4 Nov 2016 11:30:01 -0400
Subject: [PATCH 5295/5855] drm/amdkfd: Change pr_warn to pr_debug when init
 PeerDirect

Switch to use pr_debug instead of pr_warn to avoid message log
pollution when PeerDirect support is not available

Change-Id: I8366fab330632f144313a52ef2b99ab9cdf39bef
Signed-off-by: Serguei Sagalovitch <Serguei.Sagalovitch@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c b/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c
index 0edc652..3fb8896 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_peerdirect.c
@@ -457,7 +457,7 @@ void kfd_init_peer_direct(void)
 
 	if (!pfn_ib_register_peer_memory_client ||
 		!pfn_ib_unregister_peer_memory_client) {
-		pr_warn("amdkfd: PeerDirect interface was not detected\n");
+		pr_debug("amdkfd: PeerDirect interface was not detected\n");
 		/* Do cleanup */
 		kfd_close_peer_direct();
 		return;
-- 
2.7.4

